### PR TITLE
Explicity setting heron apiserver to run as cluster-admin

### PR DESCRIPTION
**Pull request by jerrypeng**
Since RBAC is enabled by default in Kubernetes 1.8

### CODE REVIEW

**Comment by jerrypeng** [comment]
@cckellogg can you test this out. Thanks

File: deploy/kubernetes/general/apiserver.yaml [hunk 1]
```diff
@@ -1,17 +1,51 @@
...
   replicas: 1
   template:
     metadata:
       labels:
         app: heron-apiserver
     spec:
+      serviceAccountName: heron-apiserver
       containers:
         - name: heron-apiserver
           image: heron/heron:0.16.2
```
Comment (cckellogg) [inline deploy/kubernetes/general/apiserver.yaml]: Can we update the image here too heron/heron:latest?

**Comment by cckellogg** [comment]
this file needs to be updated too (https://github.com/twitter/heron/blob/master/deploy/kubernetes/gcp/bookkeeper-apiserver.yaml)

File: deploy/kubernetes/general/apiserver.yaml (hunk 1 above)
Comment (jerrypeng) [reply deploy/kubernetes/general/apiserver.yaml]: sure

**Comment by jerrypeng** [comment]
@cckellogg thanks for the review.  I have addressed your comments

**Pull request merged**